docs/operations/common_configuration/overview.rst

```diff
@@ -0,0 +1,9 @@
+.. _common_configuration_overview:
+
+Common configuration overview
+=============================
```

Comment:
that was the plan to dump common config/settings scenarios here. Open to rename.
I'll get to this review some time today.